### PR TITLE
feature(as-5145): adds code expired page and associated tests for householder appeals

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-5145

## Description of change

Adds code expired page for HAS
## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by Constantine-KC — DISMISSED** [review]

File: packages/forms-web-app/__tests__/unit/routes/appeal-householder-decision/code-expired.test.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { get, post } = require('../router-mock');
+const {
+	getCodeExpired,
+	postCodeExpired
+} = require('../../../../src/controllers/appeal-householder-decision/code-expired');
+describe('routes/full-appeal/submit-appeal/code-expired', () => {
```
Comment (edpackard-work) [inline packages/forms-web-app/__tests__/unit/routes/appeal-householder-decision/code-expired.test.js:6]: change full-appeal/submit-appeal to appeal-householder-decision?

File: packages/forms-web-app/src/views/appeal-householder-decision/code-expired.njk [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{% extends "../../layouts/full-appeal-banner/main-no-back-link.njk" %}
```
Comment (edpackard-work) [inline packages/forms-web-app/src/views/appeal-householder-decision/code-expired.njk:1]: this should be /householder-banner/main-no-back-link.njk

File: packages/forms-web-app/src/views/appeal-householder-decision/code-expired.njk [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{% set resendCode="/full-appeal/submit-appeal/enter-code"%}
```
Comment (edpackard-work) [inline packages/forms-web-app/src/views/appeal-householder-decision/code-expired.njk:2]: is this correct routing for householder code-expired?

File: packages/forms-web-app/src/routes/appeal-householder-decision/index.js [hunk 4]
```diff
@@ -13,5 +13,6 @@ router.use(require('./enter-code'));
 router.use(require('./request-new-code'));
 router.use(require('./cannot-appeal'));
 router.use(require('./link-expired'));
+router.use(require('./code-expired'));
```
Comment (edpackard-work) [inline packages/forms-web-app/src/routes/appeal-householder-decision/index.js:16]: is this unit tested?

**Review by edpackard-work — APPROVED** [review]

**Review by edpackard-work — APPROVED** [review]

**Pull request merged**